lib/entity_cache/storage/persistent.rb

```diff
@@ -2,10 +2,11 @@ class EntityCache
   module Storage
     module Persistent
       extend Configure::Macro
-      configure :persistent_store
 
       def self.included(cls)
         cls.class_exec do
+          configure :persistent_store
```

Comment:
I'm going to file this as an issue on the configure library. It's more of a consideration of a feature rather than as issue. If it's never moved on, we can ultimately delete it.